lms/static/sass/course/layout/_courseware_preview.scss

```diff
@@ -103,6 +100,27 @@
             color: $action-primary-bg;
           }
         }
+        .exam-timer-clock {
+          background-color: transparent;
+          border: 1px solid $white;
+          a, b {
+            color: $white;
+          }
+          a {
+            // Overrides:
```

Comment:
@stv I thought it might be useful to be clear about the purpose of that styling, but I can remove it if you think it's excessive.